CRM/Core/SelectValues.php

```diff
@@ -1124,4 +1124,38 @@ public static function getDashboardEntriesCount() {
     return $optionValues;
   }
 
+  /**
+   * Dropdown options for quicksearch in the menu
+   *
+   * @return array
+   */
+  public static function quicksearchOptions() {
+    $includeEmail = civicrm_api3('setting', 'getvalue', array('name' => 'includeEmailInName', 'group' => 'Search Preferences'));
+    $options = [
```

Comment:
Isn't it currently possible to add to the list / alter via hook - albeit sadly by overriding the tpl too I think

Reply:
No there is currently no hook. [This extension](https://lab.civicrm.org/extensions/quicksearch) goes to great lengths to make the options configurable but it requires a lot of hacks (including javascript hacks to the menu, api wrappers, and a separate settings page).

Reply:
OK - we probably SHOULD have a hook - I think adding contribution id is a kinda common one but I can see it's not necessary within this round